dev-python/backports-functools-lru-cache/backports-functools-lru-cache-1.5.ebuild

```diff
@@ -15,36 +15,39 @@ SRC_URI="mirror://pypi/${PN:0:1}/${MY_PN}/${MY_PN}-${PV}.tar.gz"
 
 LICENSE="MIT"
 SLOT="0"
-KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-macos ~x86-macos"
```

Comment:
Any clue why ~hppa was lost?

Reply:
Apparently this is when I started depending on jaraco-packaging. There has been a rekeywording bug for a while now but I guess few people have hppa hardware:
https://bugs.gentoo.org/show_bug.cgi?id=621282

Reply:
Ok then. I was asking since we have a few cases of hppa arch team adding their keyword to old versions instead of the newest.